cmd/rowmbinaryfinder

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+echo $(compgen -c)
```

Comment:
is compgen guaranteed to be installed? Also why have a bash script that just calls another command, why not call the command directly?

Reply:
Compgen isn't an executable so it can't be called directly. :-( It's "bash built-in" whatever that means but is only available from inside bash scripts I tried "/bin/bash compgen -c" but that produced gibberish as well
https://unix.stackexchange.com/questions/151118/understand-compgen-builtin-command